test/onnx/test_utility_funs.py

```diff
@@ -1042,6 +1042,21 @@ def forward(self, x, y, z):
         for node in graph.nodes():
             self.assertIn(node.scopeName(), expected_scope_names)
 
+        expected_torch_script_scope_names = {
+            "test_utility_funs.M::/torch.nn.modules.activation.GELU::gelu1",
+            "test_utility_funs.M::/torch.nn.modules.activation.GELU::gelu2",
+            "test_utility_funs.M::/torch.nn.modules.normalization.LayerNorm::lns.0",
+            "test_utility_funs.M::/torch.nn.modules.normalization.LayerNorm::lns.1",
+            "test_utility_funs.M::/torch.nn.modules.normalization.LayerNorm::lns.2",
+            "test_utility_funs.M::",
+        }
+
+        graph, _, _ = self._model_to_graph(
+            torch.jit.script(model), (x, y, z), input_names=[], dynamic_axes={}
```

Comment:
Is an empty input_names and dynamic_axes necessary in this cirsumstances? I am not sure.

Reply:
per discussion, could be improved by revisiting `_model_to_graph` method internals.